src/configs/singularity.rs

```diff
@@ -0,0 +1,27 @@
+use crate::config::{ModuleConfig, RootModuleConfig, SegmentConfig};
+
+use ansi_term::{Color, Style};
+use starship_module_config_derive::ModuleConfig;
+
+#[derive(Clone, ModuleConfig)]
+pub struct SingularityConfig<'a> {
+    pub symbol: Option<SegmentConfig<'a>>,
```

Comment:
I think using `symbol: SegmentConfig` with `""` as default value should be good enough, as long as you don't render the prompt conditionally.

Reply:
Done.